src/app/static/src/app/hintVersion.ts

```diff
@@ -1 +1 @@
-export const currentHintVersion = "3.5.4";
+export const currentHintVersion = "3.6.0";
```

Comment:
Arguably this isn't changing functionality so shouldn't need a new version... and if it is, shouldn't the news file be getting updated too?

Reply:
Good point, I've reverted this version change. Forgot completely about the NEWS, will note that for other PRs